index.html

```diff
@@ -20,7 +20,8 @@ <h1>Examples</h1>
         <br><br>
         <a href="packages/examples/react/index.html">React Client Example</a>
         <br><br>
-        <a href="packages/examples/angular/index.html">Angular Client Example</a>
+        Please start <b>npm run start:example:angular</b> beforehand:<br>
```

Comment:
no big deal, but better use `<code>npm run start:example:angular</code>`

Reply:
Good idea, thanks. I adjusted that ⬇️